bigtop-packages/src/common/logstash/do-component-build

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+. `dirname $0`/bigtop.bom
+
+export RELEASE=1
+export LANG="en_US.UTF-8"
+
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+fi
+
+case ${ID} in
+    ubuntu | debian)
+        sudo apt-get -y update
+        sudo apt-get install -y gnupg2
```

Comment:
I'm still not get it why we need to install things during do-components-build. All the env specific packages should be installed by bigtop_toolchain. Could you elaborate more?

Reply:
Before installing rvm, we should install GPG keys with `gpg2`. (guide from https://rvm.io/ )
In ubuntu/deian, `gpg2` is not  pre-installed.